kinde_sdk/auth/oauth.py

```
@@ -1,633 +1,721 @@
-            self.auth_url = config.get("authorization_endpoint", f"{self.host}/oauth2/auth")
-            self.token_url = config.get("token_endpoint", f"{self.host}/oauth2/token")
-            self.logout_url = config.get("end_session_endpoint", f"{self.host}/logout")
-            self.userinfo_url = config.get("userinfo_endpoint", f"{self.host}/oauth2/userinfo")
-            
-        else:
-            self.auth_url = f"{self.host}/oauth2/auth"
-            self.token_url = f"{self.host}/oauth2/token"
-            self.logout_url = f"{self.host}/logout"
-            self.userinfo_url = f"{self.host}/oauth2/userinfo"
+        try:
+            tokens_data = self.get_token(code)
```

Ruff (0.11.9)

Local variable `tokens_data` is assigned to but never used

Remove assignment to unused variable `tokens_data`

(F841)